pkgs/by-name/up/upscayl-ncnn/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  cmake,
+  fetchFromGitHub,
+  fetchzip,
+  glslang,
+  installShellFiles,
+  lib,
+  libwebp,
+  ncnn,
+  stdenv,
+  vulkan-headers,
+  vulkan-loader,
+}:
+
+# upscayl-ncnn is a fork of /pkgs/by-name/re/realesrgan-ncnn-vulkan, so the nix package is basically the same.
+stdenv.mkDerivation (finalAttrs: {
+  pname = "upscayl-ncnn";
+  version = "20240601-103425";
+
+  src = fetchFromGitHub {
+    owner = "upscayl";
+    repo = "upscayl-ncnn";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-rGnjL+sU5x3VXHnvuYXVdxGmHdj9eBkIZK3CwL89lN0=";
+    fetchSubmodules = true;
```

Comment:
This is failing in [Ofborg](https://logs.ofborg.org/?attempt_id=cdbcd30c-2707-4d67-9b8f-7335e4dd04d1&key=nixos%2Fnixpkgs.356497) as the submodules use SSH, but we should actually be able to drop this as we're using the system equivalents
```suggestion
```